.circleci/config.yml

```diff
@@ -25,10 +25,11 @@ jobs:
             curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
             sudo apt-get install git-lfs
             git lfs fetch
-      - bot/docker-build-and-push:
-          applicationName: looker
-          dockerEngineVersion: 20.10.12
-          dockerBuildOptions: --build-arg EMAIL=$EMAIL --build-arg LICENSE=$LICENSE
+      - setup_remote_docker:
```

Comment:
ma remarque sur slack était de conserver l'étape `docker-build-and-push` ici et de lui passer les params qui vont bien apres avoir récupéré l'output de `full_version`